extensions/wikia/Places/PlacesController.class.php

```diff
@@ -60,11 +60,15 @@ public function placeFromModel(){
 	 * Map center can be specified
 	 */
 	public function renderMarkers() {
+		// SUS-5128 | add Google Maps API key
+		global $egMapsGMaps3ApiKey;
```

Comment:
Let's use wgGoogleMapsKey instead

Reply:
`egMapsGMaps3ApiKey` has different values for different environments while `wgGoogleMapsKey` is fixed.

Reply:
let's address that then, egMapsGMaps3ApiKey is Maps extension specific var :)

Reply:
Tried wgGoogleMapsKey on devbox. It does not work - `egMapsGMaps3ApiKey` handles this environment.

Reply:
but then we should add the proper env overrides for wgGoogleMapsKey and have egMapsGMaps3ApiKey just be a simple reassignment—if we ever change or sunset Maps extension, tight coupling to its internals will cause confusion

Reply:
See 4ae2620